shibata added the new strings in the Japanese version to the English version. (#83)
fujitake/vantiq-related: vantiq-apps-development/docs/eng/Vantiq_ExtLifecycleManagement_SetupProcedure.md

![DeployButton](../../imgs/lcm/image5.png)

Fill in the required fields.

> If pulling the image from a private repository, enter `quay-secret`, the docker-registry secret name which is created in the previous step, into the [Image Pull Secret] field.

Reviewer: ```suggestion
In case you want to pull the image from a private repository, enter `quay-secret`, the docker-registry secret name which you have created in the previous step, into the [Image Pull Secret] field.
```
